supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,6 +10,20 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
 
+[[conversational-style]]
+== Conversational style
+
+Follow _The IBM Style Guide_ guidance of _less conversational_ style in most cases. 
+
+As needed, adjust the conversational to _fairly conversational_ for an audience of new users or _least conversational_ for API documentation and other very experienced audiences.
+
+.Example AsciiDoc: Less conversational style 
+
+----
+Get answers now. Search the documentation and posts from the Stack Overflow community, which is actively monitored.
+The jobs in a stage can't pass artifacts to each other.
```

Comment:
I'd suggest against using a contraction in this example. I know that the new ISG allows it, but I'll open an issue for discussion on whether we should add guidance to continue avoiding them or not.

I'm also unsure whether we want to use exact examples from the ISG or if we should come up with our own.

Reply:
+1 
Our own examples could avoid things that might be problematic for translation too, like colloquialisms or phrasal verbs.